appserver/batch/glassfish-batch-connector/src/main/java/fish/payara/jbatch/persistence/rdbms/JBatchJDBCPersistenceManager.java

```diff
@@ -211,8 +206,6 @@ public void init(IBatchConfig batchConfig) throws BatchContainerServiceException
     public CheckpointData getCheckpointData(CheckpointDataKey key) {
         logger.entering(CLASSNAME, "getCheckpointData", key == null ? "<null>" : key);
 
-        tryObtainTableLock();
```

Comment:
I assume this is not needed for H2 for some reason?

Reply:
This was written specifically for Derby